bin/client/main.rs

```diff
@@ -440,6 +440,40 @@ async fn run() -> std::result::Result<(), ClientError> {
                         .map_err(|e| ClientError::Other(e.to_string()))?;
                     println!("Wrote key to keyring file at {}", keyring_path.display())
                 }
+                Keys::Fetch(opts) => {
+                    let new_keys = if opts.use_host || opts.key_server.is_none() {
+                        println!("Fetching host keys from bindle server");
+                        bindle_client.get_host_keys().await?
+                    } else {
+                        // SAFETY: Can unwrap because we drop into the logic block above if it was
```

Comment:
Not a big deal, but if you wanted to avoid the `unwrap` (and avoid having to explain why it's okay), you could use something like:

```rust
let new_keys = match &opts.key_server {
    Some(url) if !opts.use_host => {
        println!("Fetching host keys from {}", url);
        get_host_keys(url).await
    }
    _ => {
        println!("Fetching host keys from bindle server");
        bindle_client.get_host_keys().await
    }
}?;
```